src/modules/dashboard/index.tsx

```diff
@@ -1,14 +1,20 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { useAuthProvider } from "@core/auth";
 
-import { Board, Item, Variant } from "@components";
+import { Board, Item, Variant, Categories, CategoriesItem } from "@components";
 
 import { StyledButton, DashboardContainer } from "./style";
 import { Redirect } from "react-router";
 
 const Dashboard = () => {
   const { user } = useAuthProvider();
   const [items, setItems] = useState<Item[]>([]);
+  const [categories, setCategories] = useState<CategoriesItem[]>([]);
```

Comment:
this is not needed since we have seperate dashboard for categories, let's make this component a sidebar for user actions